documentation/making_a_config.md

```diff
@@ -51,36 +51,62 @@ input_options:
+### 8. Filter by element groups:
+A list of periodic table groups which are of interest. The groups should be specified as a list of any of the following: `["transition metal", "post-transition metal", "metalloid", "rare earth metal", "alkali", "alkaline", "alkali earth", "chalcogen", "halogen"]`
 ```filter_by_element_groups: ["transition metal"]```
-9. Energy above hull: a float of the maximum energy above hull you would like to consider
+### 9. Energy above hull:
+a float of the maximum energy above hull you would like to consider
 ```filter_by_bulk_e_above_hull: 0.05```
-10. Band gap: This requires one or both of the minimum and maximum band gap you would like to be considered to be specified. If both are, then materials in the range will be filtered for. If only the maximum is, then anything with a band gap less than the value given will be filtered for. If only the minimum is, then anything with a band gap greater than the value given will be filtered for.
+### 10. Band gap:
+This requires one or both of the minimum and maximum band gap you would like to be considered to be specified. If both are, then materials in the range will be filtered for. If only the maximum is, then anything with a band gap less than the value given will be filtered for. If only the minimum is, then anything with a band gap greater than the value given will be filtered for.
 ```
 filter_by_bulk_band_gap:
   min_gap: 0.1
   max_gap:0.3
 ```
-11. Pourbaix stability: This is only supported for Materials Project materials as of now. It selects the Pourbaix stable materials under the conditions you specify. Conditions may be specified as a range or as a list of specific values. You may also specify a maximum decoposition energy.  Step size is the increment that will be used for the interval. If the material is stable at any of the conditions considered, it will not be filtered out. A path to an lmdb file containing Pourbaix info for your materials is required. If the file path doesnt exist, the Pourbaix info will be queried from MP as a part of the run. This does take a bit of time. Be careful of having to many workers querying to avoid being black listed.
+### 11. Pourbaix stability:
+This is only supported for Materials Project materials as of now. It selects the Pourbaix stable materials under the conditions you specify. Conditions may be specified as a range or as a list of specific values. You may also specify a maximum decoposition energy.  Step size is the increment that will be used for the interval. If the material is stable at any of the conditions considered, it will not be filtered out. A path to an lmdb file containing Pourbaix info for your materials is required. If the file path doesnt exist, the Pourbaix info will be queried from MP as a part of the run. This does take a bit of time. Be careful of having to many workers querying to avoid being black listed.
```

Comment:
Somewhat unclear, could rephrase: ` Conditions may be specified as a list of specific values, or a range with an interval.`